src/map/ai/controllers/automaton_controller.cpp

```diff
@@ -1280,7 +1275,9 @@ bool CAutomatonController::TryEnhance()
         }
     }
 
-    PAutomaton->StatusEffectContainer->ForEachEffect([&protect, &shell, &haste, &stoneskin, &phalanx](CStatusEffect* PStatus) {
+    // clang-format off
+    PAutomaton->StatusEffectContainer->ForEachEffect([&protect, &shell, &haste](CStatusEffect* PStatus)
```

Comment:
Was there a reason for stoneskin and phalanx being removed from this?

Reply:
They were warned as being unused in the scope of this lambda